bitnami/grafana-operator/Chart.yaml

```diff
@@ -30,4 +30,4 @@ maintainers:
 name: grafana-operator
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
-version: 3.4.0
+version: 3.4.1
```

Comment:
Could you increase the version in a minor instead of a patch, please? It should look like this: `3.5.0`